Lib/glyphsLib/writer.py

```diff
@@ -166,3 +164,64 @@ def dumps(obj):
     fp = UnicodeIO()
     dump(obj, fp)
     return fp.getvalue()
+
+
+NSPropertyListNameSet = (
+    # 0
+    False, False, False, False, False, False, False, False,
+    False, False, False, False, False, False, False, False,
+    # 16
+    False, False, False, False, False, False, False, False,
+    False, False, False, False, False, False, False, False,
+    # 32
+    False, False, False, False, True, False, False, False,
+    False, False, False, False, False, False, True, False,
+    # 48
+    True, True, True, True, True, True, True, True,
+    True, True, False, False, False, False, False, False,
+    # 64
+    False, True, True, True, True, True, True, True,
+    True, True, True, True, True, True, True, True,
+    # 80
+    True, True, True, True, True, True, True, True,
+    True, True, True, False, False, False, False, True,
+    # 96
+    False, True, True, True, True, True, True, True,
+    True, True, True, True, True, True, True, True,
+    # 112
+    True, True, True, True, True, True, True, True,
+    True, True, True, False, False, False, False, False
+    )
+
+
+def _needs_quotes(string):
+    if len(string) == 0:
+        return True
+    if not isinstance(string, (str, unicode)):
+        return False
```

Comment:
the argument name is "string". How can something that is called "string" not be an instance of (str, unicode)?
also, `_needs_quotes` is called below, after the very same isinstance test, so this additional check seems redundant to me. If anything should be an assert